ntoskrnl/io/pnpmgr/pnpmap.c

```diff
@@ -29,33 +36,89 @@ static UNICODE_STRING ConfigurationDataU = RTL_CONSTANT_STRING(L"Configuration D
 static UNICODE_STRING BootConfigU = RTL_CONSTANT_STRING(L"BootConfig");
 static UNICODE_STRING LogConfU = RTL_CONSTANT_STRING(L"LogConf");
 
-/* FIXME: There should be two sets of hardcoded PnP identifiers
- * for the keyboard and pointer peripherals (see CORE-18963).
- * They also can be parsed from a LegacyXlate sections of driver INF files.
- */
+/* FIXME: Trailing \0 in structures below are hacks, should be removed.
```

Comment:
An identifier string will always have a length of 7

Reply:
> 19:46 Stas'M:
This is needed for the underlying code which works with multi-line registry strings